app/models/ability.rb

```diff
@@ -210,6 +210,10 @@ def signed_in_with_cfp_role(user)
       role.resource_type == 'Conference' && role.name == 'cfp' &&
       (Conference.with_role(:cfp, user).pluck(:id).include? role.resource_id)
     end
+
+    can [:index, :revert_object, :revert_attribute], PaperTrail::Version do |version|
+      %w(Event Commercial).include?(version.item_type) && (conf_ids_for_cfp.include? version.conference_id)
```

Comment:
That would allow users with cfp role to also edit Commercials of conference. You should provide a quick fix for this here, and let's discuss a bit about improving the authorization of versions in general. I opened #1144